File: exercises-python/2. Problemas de estructura condicional/29.py
'''
29.Una empresa calcula el sueldo bruto de sus trabajadores
en base a las horas trabajadas. Hasta 48 horas, se paga una
tarifa horaria normal. Para las horas extras se paga un recargo
del 20%. Por otro lado, si el sueldo bruto es superior a S/. 1500
se aplica un descuento del 11%. Desarrolle el programa que muestre
las horas trabajadas, el pago por hora, el sueldo bruto, el descuento
y el total a pagar por la empresa a sus trabajadores.
'''

import os
os.system("cls")

horas = int(input("Horas Trabajadas: "))
tarifaHoraria = int(input("Taria Horaria: "))
descuento = 0

if horas < 48:
    sueldoBruto = tarifaHoraria * horas
else:
    sueldoBruto = (tarifaHoraria * 48)
    sueldoBruto += sueldoBruto * 0.20

if sueldoBruto > 1500:
    descuento = sueldoBruto * 0.11
total = sueldoBruto - descuento

print(f"")
print(f"Pago por Horas: {tarifaHoraria}")
print(f"Sueldo Bruto: {sueldoBruto}")
print(f"Descuento: {descuento}")
print(f"Total: {total}")

File: exercises-cpp/3. C++/2. Condicionales/29.cpp
 #include <iostream>
#include <cstdio>
#include <strings.h>
#include <conio.h>
using namespace std;

int main(){
    system("cls");

    int horas,tarifaHoraria,descuento,sueldoBruto,total;
    printf("Hora Trabajadas: "); cin>>horas;
    printf("Taria Horaria: "); cin>>tarifaHoraria;
    if (horas < 48){
        sueldoBruto = tarifaHoraria * horas;
    }
    else{
        sueldoBruto = (tarifaHoraria * 48);
        sueldoBruto += sueldoBruto * 0.20;
    }
    if (sueldoBruto > 1500){
        descuento = sueldoBruto * 0.11;
    }
    total = sueldoBruto - descuento;

    
    cout<<"Pago por Horas: "<<tarifaHoraria<<endl;
    cout<<"Sueldo Bruto: "<<sueldoBruto<<endl;
    cout<<"Descuento: "<<descuento<<endl;
    cout<<"Total: "<<total<<endl;

}